Dshop/apps/products_catalogue/templates/products_catalogue/product_detail.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<div class="container mt-5 mb-5">
+    <div class="row d-flex justify-content-center">
+        <div class="col-md-10">
+            <div class="card">
+                <div class="row">
+                    <div class="col-md-6">
+                        <div class="images p-3">
+                            {% for pic in product.images.all %}
```

Comment:
does this work? DetailView returns "object". Check DetailView here: https://akademiait.com.pl/class-based-views-w-django-wygodny-sposob-tworzenia-widokow-czesc-1/